### PR TITLE
Indentation fixes

### PR DESCRIPTION
**Pull request by locus313**
After this commit:
https://github.com/azerothcore/azerothcore-wotlk/commit/d2cc3fcbc2d7bf0b185eb03d169afc9783870568

Docker build is failing with indentation errors. These updates resolved the errors for me.

Tested on Centos 7

### CODE REVIEW

File: src/AutoBalance.cpp [hunk 1]
```diff
@@ -48,8 +48,8 @@
 bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
     bool ret=true;
     FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
-        if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
-            ret=false; // we change ret value only when scripts return false
+    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
+        ret=false; // we change ret value only when scripts return false
```
Comment (Helias) [inline src/AutoBalance.cpp:52]: add curly brackets

File: src/AutoBalance.cpp [hunk 2]
```diff
@@ -709,8 +709,8 @@ class AutoBalance_AllCreatureScript : public AllCreatureScript
             else {
                 newDmgBase=creatureStats->BaseDamage[2];
                 // special increasing for end-game contents
-                if (LevelEndGameBoost && !creature->GetMap()->IsRaid())
-                    newDmgBase *= creatureABInfo->selectedLevel >= 75 && originalLevel < 75 ? float(creatureABInfo->selectedLevel-70) * 0.3f : 1;
+            if (LevelEndGameBoost && !creature->GetMap()->IsRaid())
+                newDmgBase *= creatureABInfo->selectedLevel >= 75 && originalLevel < 75 ? float(creatureABInfo->selectedLevel-70) * 0.3f : 1;
```
Comment (Helias) [inline src/AutoBalance.cpp:713]: add curly brackets

**Comment by Helias** [comment]
Add always curly brackets {} when you can, I strongly recommend you to AVOID the inline instructions statement

File: src/AutoBalance.cpp (hunk 2 above)
Comment (locus313) [reply src/AutoBalance.cpp:717]: @Helias also got this one done. Let me know if that looks good now.

File: src/AutoBalance.cpp (hunk 1 above)
Comment (Helias) [reply src/AutoBalance.cpp:53]: there is a problem about indentation.
WRONG:
```C++
bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
    bool ret=true;
    FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
        {
        ret=false; // we change ret value only when scripts return false
        }
    return ret;
}
```
RIGHT:
```C++
bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
    bool ret=true;
    FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount)) {
        ret=false; // we change ret value only when scripts return false
    }
    return ret;
}
```

Comment (locus313) [reply src/AutoBalance.cpp:52]: @Helias Thanks. just updated.

**Pull request merged**